examples/options/src/track_propagation.cpp

```diff
@@ -30,10 +30,14 @@ track_propagation::track_propagation()
                          po::value(&(config.navigation.overstep_tolerance))
                              ->default_value(-100.f),
                          "The overstep tolerance [um]");
+    m_desc.add_options()("min-mask-tolerance-mm",
+                         po::value(&(config.navigation.min_mask_tolerance))
+                             ->default_value(1e-5f),
```

Comment:
This value worked pretty well so far in detray. It should be related to the accumulated path * mean relative propagation error. So according to https://cds.cern.ch/record/1113528/files/ATL-SOFT-PUB-2009-001.pdf (19) for a ~10cm long track at an rk error tolerance of 10^(-4):
```
100mm * 2.2 * 10^(-5) * (10^(-4))^(0.57)
```
which is pretty much ```10^(-5)```, I believe. Or did I get this wrong? Don't know if it is meant to be used for single precision for example

Reply:
If it is selected based on the literature then yes.. :/ 

btw, It is a bit suspicious that a power is applied to length ($\tau$)

Reply:
Yeah, the fit is a bit weird. But it works quite well so far. However, this is only applicable for shapes where every boundary is a length

Reply:
Apart from that, we might want to grant a larger value to the mask tolerance because there can be errors not only from the RK itself but also from the parameter uncertainty. But we don't know the empirically good value -- so we can just go ahead with this for the moment